src/core.js

```diff
@@ -2,32 +2,48 @@
 const person = { age: 32, size: 'Small' }
 
 // 1. Give the person object a name property with the value Matt
-
+;(person.name = 'Matt'), console.log(person)
 // 2. Remove the size property from the person
-
+delete person.size
+console.log(person)
 // 3. Increase the person's age by 11
-
+person.age = person.age + 11
+console.log(person)
 // 4. Add an instruments property to the person, initialised as an empty array
-
+person['instruments']
```

Comment:
this doesn't look correct, are you sure it's working correctly and that `person.instruments` is an empty array `[]`?